client/dashboard/sites/overview-card/upsell-with-action.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+	Button,
+	Card,
+	CardBody,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { ComponentProps } from 'react';
+import { useAnalytics } from '../../app/analytics';
+import ComponentViewTracker from '../../components/component-view-tracker';
+import { upsell } from '../../components/icons';
+import type { ReactNode } from 'react';
+
+interface UpsellWithActionCardProps {
+	action: ComponentProps< typeof Button > & { href: string };
```

Comment:
Since an anchor button is mandatory ...

```suggestion
	action: ButtonAsAnchorProps; // imported from '@wordpress/components/build-types/button/types'
```